security/symmetric/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "azure-iot-security-symmetric-key",
+  "version": "1.0.0",
+  "description": "Azure IoT Symmetric Key Security Client",
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "main": "index.js",
+  "typings": "index.d.ts",
+  "dependencies": {
+    "azure-iot-common": "1.8.0",
+    "debug": "^3.0.1",
+    "base32-encode": "^1.0.0",
+    "machina": "^2.0.1",
+    "tss.js": "^0.0.3",
+    "array-equal": "^1.0.0"
```

Comment:
some of those aren't actually needed.

Reply:
Fixed.

---
In reply to: [231364785](https://github.com/Azure/azure-iot-sdk-node/pull/418#discussion_r231364785) [](ancestors = 231364785)